src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -44,6 +44,15 @@ internal class DaprHttpInteractor : IDaprInteractor
             this.httpClient = this.CreateHttpClient();
         }
 
+        public DaprHttpInteractor(
+            HttpMessageHandler clientHandler,
+            string httpEndpoint,
+            string apiToken,
+            TimeSpan? requestTimeout) : this(clientHandler, httpEndpoint, apiToken)
+        {
+            this.httpClient.Timeout = requestTimeout ?? this.httpClient.Timeout;
+        }
```

Comment:
Does this need to be its own constructor? The existing one is internal.

Reply:
Removed it. Didn't think about the fact that since it's internal it can't be referenced and thus won't break anyone's code when I changed the signature.